Formula/tmux-xpanes.rb

```diff
@@ -0,0 +1,18 @@
+class TmuxXpanes < Formula
+  desc "Ultimate terminal divider powered by tmux"
+  homepage "https://github.com/greymd/tmux-xpanes"
+  url "https://github.com/greymd/tmux-xpanes.git", :tag => "v2.2.1"
+  head "https://github.com/greymd/tmux-xpanes.git", :branch => "master"
+  depends_on "tmux" => :recommended
```

Comment:
Should be a new line between `head` and the first `depends_on` here.

Reply:
Fixed in 80bc91c